Gemfile

```diff
@@ -12,11 +15,17 @@ gem 'sqlite3'
 
 gem 'sprockets-rails'
 
+group :development do
+  gem 'brakeman'
+end
+
 group :development, :test do
   gem 'rubocop', require: false
 end
 
-gem 'overcommit'
+gem 'bundler-audit', require: false
```

Comment:
@GowthamK01 Please check if we can add this only to the `development` group

Reply:
@gomathi-mallow Checked and added the bundler-audit inside the `development` group